src/onnx/parse_dynamicquantizelinear.cpp

```diff
@@ -139,10 +139,25 @@ struct parse_dynamicquantizelinear : op_parser<parse_dynamicquantizelinear>
         auto y_zero_point = info.add_instruction(
             migraphx::make_op("convert", {{"target_type", migraphx::shape::uint8_type}}), round);
 
+        // 2.5 Adjust so output is int8 instead of uint due to MLIR being unable to handle int8 *
```

Comment:
Question. Couldn't the previous step to round etc. be directly giving out int8? Thanks.

Reply:
No, unfortunetly the output from additional onnx tests as part of the standard dyanicQuantizelinear needs to always output uint8 here. That seems to be what's also failing in CI right now eg)

```

 ACTUAL: dtype('uint8')
 DESIRED: dtype('int8')

======================================================================
FAIL: test_dynamicquantizelinear_min_adjusted_cpu (__main__.OnnxBackendNodeModelTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspace/AMDMIGraphX/build_602/test/py/venv-onnx/3.10/lib/python3.10/site-packages/onnx/backend/test/runner/__init__.py", line 290, in device_test_func
    return test_func(*args, device=device, **kwargs)
  File "/workspace/AMDMIGraphX/build_602/test/py/venv-onnx/3.10/lib/python3.10/site-packages/onnx/backend/test/runner/__init__.py", line 467, in run
    self.assert_similar_outputs(
  File "/workspace/AMDMIGraphX/test/py/onnx_backend_test.py", line 51, in assert_similar_outputs
    np.testing.assert_equal(ref_outputs[i].dtype,
  File "/workspace/AMDMIGraphX/build_602/test/py/venv-onnx/3.10/lib/python3.10/site-packages/numpy/testing/_private/utils.py", line 425, in assert_equal
    raise AssertionError(msg)
AssertionError: 
Items are not equal:

```